app/views/application/_messages.html.erb

```diff
@@ -1,6 +1,3 @@
 <% if flash[:notice] %>
   <p class="flash-notice"><%= flash[:notice] %></p>
 <% end %>
-<% if flash[:error] %>
```

Comment:
Does this harm anything? As a developer working on the application if flash messages had the ability to be displayed, I'd expect the standard categories to display without too much extra work (error, notice, warning). Removing this feels like it'd just surprise Future Me.